warp10/src/main/java/io/warp10/script/functions/TOLONGBYTES.java

```diff
@@ -49,13 +50,47 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
         } else {
           stack.push(Longs.toByteArray(((Long) o).longValue()));
         }
+      } else if (o instanceof List && nbBytes > 0 && nbBytes <= 8) {
+        try {
+          ByteBuffer bytes = ByteBuffer.allocate(nbBytes * ((List) o).size());
+          if (8 == nbBytes) {
+            for (Object el : (List) o) {
+              bytes.putLong((Long) el);
```

Comment:
You are not checking that the elements are indeed Longs

Reply:
No, that would be a time waste if there is one million long. That's why I put it in a try/catch.

Reply:
ok